### PR TITLE
Feature/login & signup page

### PR DESCRIPTION
**Pull request by swappnet**
There is a problem with login (look Auth.tsx), registration works fine

### CODE REVIEW

File: apps/web/pages/index.tsx [hunk 1]
```diff
@@ -17,7 +17,6 @@ export default function Web() {
       </Head>
       {/* Put here already styled Home page component & remove Header! */}
       <Header variant="welcome" userName={user ? user?.username : 'Guest'} />
-      <Input variant="search" />
     </DynamicLayout>
   );
 }
```
Comment (Gw3i) [inline apps/web/pages/index.tsx:22]: If I am not logged in, I should see the login or be at least redirected to the login page

**Review by Gw3i — CHANGES_REQUESTED** [review]

File: apps/web/pages/index.tsx (hunk 1 above)
Comment (swappnet) [reply apps/web/pages/index.tsx:22]: I fix that, thanks

Comment (swappnet) [reply apps/web/pages/index.tsx:25]: Done

**Review by Joshibbotson — APPROVED** [review]
Nice, you've addressed the changes.

**Review by Gw3i — APPROVED** [review]

**Pull request merged**